app/controllers/concerns/audited_reports/process_control_stats.rb

```diff
@@ -0,0 +1,350 @@
+
+    column_data                   = []
+    column_widths, column_headers = [], []
+
+    @columns.each do |col_name, col_title, col_width|
+      column_headers << "<b>#{col_title}</b>"
+      column_widths  << pdf.percent_width(col_width)
+    end
+
+    if @user_process_control_data
+      @user_process_control_data[:process_control_data].each do |row|
+        new_row = []
+
+        @columns.each do |col_name, _|
+          new_row << (row[col_name].kind_of?(Array) ?
+                      row[col_name].map {|l| "  • #{l}"}.join("\n") :
+                      row[col_name])
+        end
+
+        column_data << new_row
+      end
+
+      unless column_data.blank?
+        pdf.text @business_unit_title
+
+        pdf.move_down PDF_FONT_SIZE
+
+        pdf.font_size((PDF_FONT_SIZE * 0.75).round) do
+          table_options = pdf.default_table_options(column_widths)
+
+          pdf.table(column_data.insert(0, column_headers), table_options) do
+            row(0).style(
+              :background_color => 'cccccc',
+              :padding => [(PDF_FONT_SIZE * 0.5).round, (PDF_FONT_SIZE * 0.3).round]
+            )
+          end
+        end
+
+        pdf.move_down PDF_FONT_SIZE
+        pdf.text [
+          Review.model_name.human(count: 0),
+          @user_process_control_data[:review_identifications].to_sentence
+        ].join(': ') , inline_formati: true
+      else
+        pdf.text(
+          t("#{@controller}.process_control_stats.without_reviews_in_the_period"))
+      end
+    end
+
+    pdf.move_down PDF_FONT_SIZE
+
+    column_data = []
+
+    if @process_control_data
+      @process_control_data[:process_control_data].each do |row|
+        new_row = []
+
+        @columns.each do |col_name, _|
+          new_row << (row[col_name].kind_of?(Array) ?
+                      row[col_name].map {|l| "  • #{l}"}.join("\n") :
+                      row[col_name])
+        end
+
+        column_data << new_row
+      end
+
+      unless column_data.blank?
+        pdf.text @business_unit_type_title
+
+        pdf.move_down PDF_FONT_SIZE
+
+        pdf.font_size((PDF_FONT_SIZE * 0.75).round) do
+          table_options = pdf.default_table_options(column_widths)
+
+          pdf.table(column_data.insert(0, column_headers), table_options) do
+            row(0).style(
+              :background_color => 'cccccc',
+              :padding => [(PDF_FONT_SIZE * 0.5).round, (PDF_FONT_SIZE * 0.3).round]
+            )
+          end
+        end
+
+        pdf.move_down PDF_FONT_SIZE
+        pdf.text t(
+          "#{@controller}.process_control_stats.review_effectiveness_average",
+          score: @process_control_data[:reviews_score_data]
+        ), inline_format: true
+
+        pdf.move_down PDF_FONT_SIZE * 0.25
+
+        pdf.text [
+          Review.model_name.human(count: 0),
+          @process_control_data[:review_identifications].to_sentence
+        ].join(': ') , inline_formati: true
```

Comment:
Esta opción debería ser:
```suggestion
        ].join(': ') , inline_format: true
```